ansible/roles/repository/templates/Jenkinsfile.j2

```diff
@@ -1210,6 +1214,7 @@ pipeline {
                 -e WEB_AUTH=\"${CI_AUTH}\" \
                 -e WEB_PATH=\"${CI_WEBPATH}\" \
                 -e NODE_NAME=\"${NODE_NAME}\" \
+                -e SYFT_IMAGE_TAG=\"${SYFT_IMAGE_TAG}\" \
```

Comment:
Maybe we should consider something like `${CI_SYFT_IMAGE_TAG:${SYFT_IMAGE_TAG}}` and give ourselves the option to supply different values if required for the two processes.